src/components/molecules/FilterContent/FilterContent.test.tsx

```diff
@@ -3,7 +3,6 @@ import { describe, it, expect, vi } from 'vitest';
 import { FilterContent, type ContentTab } from './FilterContent';
 import { CONTENT } from '@/core/stores/filters/filters.types';
 
-// Mock libs - use actual utility functions and icons from lucide-react
```

Comment:
no need to remove this comment which is a reminder to use actual library functions including upstream dependency libraries (especially important for snapshot testing).

If you need to mock libs for specific reason then best done for individual tests